data/src/main/java/dk/nodes/template/data/network/RestPostRepository.kt

```diff
@@ -7,38 +7,31 @@ import kotlinx.coroutines.channels.BroadcastChannel
 import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.asFlow
+import retrofit2.Response
 import javax.inject.Inject
 
 class RestPostRepository @Inject constructor(private val api: Api) : PostRepository {
 
-    private val postsChannel = BroadcastChannel<List<Post>>(Channel.CONFLATED)
-        .also {
-            it.offer(listOf())
-        }
+    private val postsChannel = BroadcastChannel<List<Post>>(Channel.CONFLATED).apply {
+        offer(listOf())
+    }
```

Comment:
Can be reduced even more replacing it with` ConflatedBroadcastChannel<List<Post>>(listOf())`

Reply:
Sweet!